relay/nakama/allowlist.go

```diff
@@ -132,41 +132,41 @@ type ClaimKeyRes struct {
 	Success bool `json:"success"`
 }
 
-func claimKeyRPC(ctx context.Context, _ runtime.Logger, _ *sql.DB, nk runtime.NakamaModule, payload string) (
+func claimKeyRPC(ctx context.Context, logger runtime.Logger, _ *sql.DB, nk runtime.NakamaModule, payload string) (
 	string, error,
 ) {
 	userID, err := getUserID(ctx)
 	if err != nil {
-		return "", err
+		return logCode(logger, Internal, "unable to get userID")
```

Comment:
We're losing the contents of the error here. It can be surfaced to the client with:
`logCode(logger, Internal, "unable to get userID: %v", err)`

If we don't want to surface the specific error reason, we can just log it here (or extend logCode to take in an error that will just be logged then discarded).

Reply:
went with surfacing it